test/test-conn.py

```diff
@@ -202,7 +202,12 @@ def handle_register(msg):
     sensor['sock'] = s
```

Comment:
in the commit 386ff07958b32bb6aa994d1fc2bd5e7dd2263bed:

Resend auth message if receive an error in script test-conn

Reply:
done